packages/core/admin/admin/src/content-manager/components/AttributeFilter/AdminUsersFilter.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+import { Combobox, ComboboxOption } from '@strapi/design-system';
+import { getDisplayName } from '@strapi/helper-plugin';
+import PropTypes from 'prop-types';
+import { useIntl } from 'react-intl';
+
+import { useAdminUsers } from '../../../hooks/useAdminUsers';
+
+const AdminUsersFilter = ({ value, onChange }) => {
+  const { formatMessage } = useIntl();
+  const { users, isLoading } = useAdminUsers({}, { staleTime: 2 * (1000 * 60) });
```

Comment:
I'm not sure about `staleTime` here. The value seems rather arbitrary. Could you explain your thinking behind it?

Reply:
Haha, ok. Let's remove it here. react-query takes care of the caching for us during rendering, and once the list of users needs to be fetched, it should be fresh (e.g. because a new user has been added since the last load).

The CM fetches more than it should in general, but therefore always delivers non-stale data, which IMO is preferred for the list-views.